### ISSUE TITLE
Language "Use the system default" is no longer working, falls back to English (Fedora, KDE)

### ISSUE THREAD

**Issue by magicgoose** [strawberrymusicplayer/strawberry#1570]
- [x] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [x] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [x] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
Language "Use the system default" is no longer working as expected - instead, Strawberry uses English ignoring the system-wide language.

**To Reproduce**
Use Fedora KDE Spin.
Set language to non-English, for example Dutch (= Nederlands).
Install Strawberry Fedora package.
Make sure Strawberry Language = "Use the system default", restart if needed.

**Expected behavior**
Strawberry to use system language. (this worked before the latest few weeks)

**Screenshots:**
![afbeelding](https://github.com/user-attachments/assets/d37b9365-4b08-48ce-923d-0c6de66cb334)

**System Information:**
 - Operating system: Fedora Linux 40
 - Strawberry Version: 1.1.3

**Additional context**
 - Graphics Platform: Wayland
 - Qt Version: 6.7.2
 - KDE Plasma Version: 6.1.5
 - KDE Frameworks Version: 6.6.0
 - Kernel Version: 6.10.11-200.fc40.x86_64 (64-bit)

Most other applications (which are translated to begin with) are still translated on this system, everything indicates that something specific to Strawberry has recently happened.

**Comment by magicgoose** [comment]
> I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed

from ChangeLog/Unreleased:
>    * Require Qt 6.4 or higher and drop support for Qt 5.
>    * Use Qt stringliterals.

could any of this possibly constitute a fix? I'm not sure

**Comment by jonaski** [comment]
I renamed all the .po files when I merged translations to Crowdin, that probably has something to do with it.

**Comment by jonaski** [comment]
What is `LC_MESSAGES` set to?
Check with `locale | grep LC_MESSAGES`

**Comment by jonaski** [comment]
Is suspect this is a Qt bug (https://bugreports.qt.io/browse/QTBUG-129434), I see on Windows that it returns "en_US" before the language set as the display language, I think the order of the languages are wrong, since strawberry only uses the first.
I don't know if it's a good idea parse and try all.

**Comment by magicgoose** [comment]
> What is `LC_MESSAGES` set to? Check with `locale | grep LC_MESSAGES`

    ~> locale | grep LC_MESSAGES
    LC_MESSAGES="nl_NL.UTF-8"

**Comment by fvalenduc** [comment]
I have gentoo and I have the same problem since strawberry-1.1.2. LC_MESSAGES is set to fr_BE.utf8

**Comment by jonaski** [comment]
Please test with 6.8.0 and check if the issue is gone.

**Comment by fvalenduc** [comment]
I have tested strawberry 1.2.0 and qt 6.8.0 in Archlinux and the problem still occurs.

**Comment by jonaski** [comment]
According to the changelog for 6.8.1 there are still fixes for wrong language issues in that release too: https://code.qt.io/cgit/qt/qtreleasenotes.git/about/qt/6.8.1/release-note.md
So please try 6.8.1 once available in your distro, I still suspect this is a Qt bug.

**Comment by fvalenduc** [comment]
Still the same with QT 6.8.1 in archlinux.

**Comment by magicgoose** [comment]
@jonaski yes it's working now, thanks for fixing!

**Comment by jonaski** [comment]
I've added fr_BE, just copied the fr_FR translations.

**Comment by magicgoose** [comment]
it's happening again, on 1.2.14 (Qt 6.9.3)

with the system language setting, some buttons are localized (OK, Apply, Cancel) but the rest of the UI isn't.

I can't reopen, should I create a new ticket? @jonaski

**Comment by jonaski** [comment]
There is a new fix for this in 1.2.15 (I rewrote much of the code to try all UI language codes instead of just the first), please try it first: https://github.com/strawberrymusicplayer/strawberry/releases/tag/1.2.15

**Comment by fvalenduc** [comment]
Indeed, it works correctly when choosing the system language in 1.2.15
